### ISSUE TITLE
Update plugin description to suggest disabling adblock (otherwise upload of json private key may fail)

### ISSUE THREAD

**Issue by lieszkol** [rainlab/googleanalytics-plugin#105]
I ran into the same issue that one reviewer mentioned on the latest review:

> Tom Steventon Found the plugin not useful on 26 Aug, 2020
> Does not appear to work currently. Pops up a message simply saying "error" when trying to add the json key file.

At first I went down the rabbit hole and though somehow .json extension is being blocked, because I wasn't able to upload such a file via the media manager either. So I thought I may need to add something like this to /config/cms.php:

```
 'fileDefinitions' => [
        'assetExtensions' => array_merge(\October\Rain\Filesystem\Definitions::get('assetExtensions'), ['json']),
    ],
```

But still no luck. 

Finally it dawned on me that my adblocker (Chrome uBlock origin) may be interfering. Disabling that fixed the issue and I was subsequently able to upload the json private key.

I think this should be highlighted in the plugin description so others don't give up on it or rate it as "not useful".

**Comment by LukeTowers** [comment]
@lieszkol feel free to submit a PR with such a notice

**Comment by lieszkol** [comment]
Ok, done: https://github.com/rainlab/googleanalytics-plugin/pull/106

**Comment by bennothommo** [comment]
Fixed by #106.
